docs/source/faq/index.rst

```diff
@@ -158,7 +158,7 @@ including the following commands in your notebook before creating any plots:
     import plotly.offline as po
     po.init_notebook_mode(connected=True)
 
-Note that FiftyOne currently requires `plotly>=4.14,<5`, which should have been
+Note that FiftyOne currently requires `plotly`, which should have been
```

Comment:
Since we no longer have an upper bound requirement on `plotly`, perhaps this paragraph can just be deleted?

Reply:
Sounds good will update